### ISSUE TITLE
Feature Request: Allow exporting of all documents

### ISSUE THREAD

**Issue by mynameismon** [hackmdio/hackmd-cli#48]
It would be really helpful if one could export all documents with a flag.

For example:
```
hackmd export --all --md
```

This way, one can easily keep their documents on their device as a backup (in case any loss of data occurs)

**Comment by morganpartee** [comment]
Agreed - Publish doesn't meet my needs so I need a way to fetch all docs 

(or only changed docs, preferably... but I can dream haha.)
